app/models/post.rb

```diff
@@ -49,6 +49,8 @@ class Post < ApplicationRecord
   belongs_to :spoiled_unit, polymorphic: true
   has_many :post_likes, dependent: :destroy
   has_many :comments, dependent: :destroy
+  # TODO: what happens if the original is destroyed?
```

Comment:
oh no, this is my question. Are we just going to go with if the original is destroyed, destroy all shared posts + comments on those shared posts?